terraform/common/data.tf

```diff
@@ -0,0 +1,11 @@
+data aws_iam_group administrators {
```

Comment:
Can we create the groups via terraform as well?
Also the group names may look nicer starting with a capital letter.